src/data/situations_de.json

```diff
@@ -0,0 +1,96 @@
+{
+    "section-main": {
+      "headline": {
+        "title": "Clustersituationen",
+        "textblock": [
+          "Du kannst für folgende Situationen ein erhöhtes Ansteckungsrisiko annehmen.",
+          "Wenn du sie nicht vermeiden kannst, nutze die Corona-Warn-App, um im Falle einer nachträglich erkannten SARS-CoV-2-Infektionskette schnell gewarnt zu werden, oder selbst warnen zu können.",
+          "Basierend auf einer Analyse gemeinsamer Eigenschaften von Infektionsclusters, <a href=\"https://www.mhlw.go.jp/content/10900000/000635891.pdf\">entwickelte Japan ein Konzept names \"Three Cs\"</a> entwickelt, um Hochrisiko-Orte und -Situationen zu erkennen. Übersetzt ins Deutsche lautet es:"
+        ]
+      }
+    },
+    "section-general-listing": {
+      "headline": {
+        "title": "Die Fünf Gs",
```

Comment:
Maybe this wording is too in-your-face for some people who we also need to convince?